Polish README.md, adding project description and making instructions …

riley-1995 opened the pull request:
…more concise.

madisonewebb reviewed (approved):
Love it! Thank you for doing this! Do we want to include a link to the live website? (Which was broken until now!! My ISP changed my IP for the first time since I moved and I didn't know!)

riley-1995 commented:
> Love it! Thank you for doing this! Do we want to include a link to the live website? (Which was broken until now!! My ISP changed my IP for the first time since I moved and I didn't know!)

Just added one!

smartinez1123 reviewed (approved):
Looks good to me.
